datasets/glossis/glossis.py

```diff
@@ -60,38 +60,40 @@
         "prefix", type=str, nargs=1, help="Input folder/prefix", default="fews_glossis/"
     )
     parser.add_argument("assetfolder", type=str, nargs=1, help="GEE asset")
-
+    # 
+    parser.add_argument("gee_bucket_folder", type=str, nargs=1, help="GEE bucket folder")
     # TODO: change all these sections to separate commands and make sure they run independent
     # instead of creating one script to rule them all...
     parser.add_argument(
-        "--skip-waterlevel", dest="skip_waterlevel", default=False, action="store_true"
+        "--waterlevel", dest="waterlevel", default=False, action="store_true"
     )
     parser.add_argument(
-        "--skip-wind", dest="skip_wind", default=False, action="store_true"
+        "--wind", dest="wind", default=False, action="store_true"
     )
     parser.add_argument(
-        "--skip-currents", dest="skip_currents", default=False, action="store_true"
+        "--currents", dest="currents", default=False, action="store_true"
     )
     parser.add_argument(
-        "--skip-waves", dest="skip_waves", default=False, action="store_true"
+        "--waves", dest="waves", default=False, action="store_true"
     )
     parser.add_argument(
-        "--skip-flowmap-tiffs", dest='skip_flowmap_tiffs', default=False, action='store_true'
+        "--flowmap-tiffs", dest='flowmap_tiffs', default=False, action='store_true'
     )
     parser.add_argument(
-        "--skip-flowmap-tiles", dest='skip_flowmap_tiles', default=False, action='store_true'
+        "--flowmap-tiles", dest='flowmap_tiles', default=False, action='store_true'
     )
     parser.add_argument(
-        "--flowmap-history", dest='flowmap_history', default=False, action='store_true'
+        "--skip-flowmap-history", dest='skip_flowmap_history', default=False, action='store_true'
```

Comment:
Why also invert this one? Maybe also put the default to True?

Reply:
I'll remove this argument it is not usefull anymore.